contrib/python/scripts/scale_viscous_flow_law_prefactors_aspect.py

```diff
@@ -0,0 +1,287 @@
+# This script scales the prefactors terms from viscous flow laws commonly used
+# in geodynamic modeling in a manner that is internally consistent with ASPECT's
+# conventions following Dannberg et al. (2017, https://doi.org/10.1002/2017GC006944).
+
+import numpy as np
+
+# Gleason and Tullis (1995) - Wet Quartzite - Dislocation Creep
+
+pfct_gt95_wqtz_disl_rprt = 1.8e-8 # Original reported prefactor in units of MPa^-n micrometers^m
+
+sexp_gt95_wqtz_disl_rprt = 4.0    # Original reported stress exponent
+
+gexp_gt95_wqtz_disl_rprt = 0      # Original reported grain size exponent
+
+# Convert prefactor to units of Pa^-n meters^m s^-1
+
+pfct_gt95_wqtz_disl_unit = (pfct_gt95_wqtz_disl_rprt) * \
+                           (1.e-6**sexp_gt95_wqtz_disl_rprt) * \
+                           (1.e-6**gexp_gt95_wqtz_disl_rprt)
+
+# Calculate prefactor scaling term
+
+pfct_gt95_wqtz_disl_sfac = 2.**(sexp_gt95_wqtz_disl_rprt-1.) * 3.**((sexp_gt95_wqtz_disl_rprt+1.)/2.)
+
+# Calculated modified prefactor term for ASPECT
+
+pfct_gt95_wqtz_disl_aspt = pfct_gt95_wqtz_disl_unit * pfct_gt95_wqtz_disl_sfac
+
+print('')
+
+print('Gleason and Tullis (1995) Quarzite w/ Melt Dislocation Prefactor Scaling = ', pfct_gt95_wqtz_disl_sfac)
+
+print('Gleason and Tullis (1995) Quarzite w/ Melt Dislocation Prefactor ASPECT  = ', pfct_gt95_wqtz_disl_aspt)
+
+print('')
+
+
+# Gleason and Tullis (1995) - Dry Quartzite - Dislocation Creep
+
+pfct_gt95_dqtz_disl_rprt = 1.1e-4 # Original reported prefactor in units of MPa^-n micrometers^m
+
+sexp_gt95_dqtz_disl_rprt = 4.0    # Original reported stress exponent
+
+gexp_gt95_dqtz_disl_rprt = 0      # Original reported grain size exponent
+
+# Convert prefactor to units of Pa^-n meters^m s^-1
+
+pfct_gt95_dqtz_disl_unit = (pfct_gt95_dqtz_disl_rprt) * \
+                           (1.e-6**sexp_gt95_dqtz_disl_rprt) * \
```

Comment:
Might be worth making a variable called MPa_to_Pa and micrometer_to_meter to improve readability.